Products/Archetypes/skins/archetypes/widgets/js/textcount.js

```diff
@@ -1,12 +1,13 @@
 <!-- Original:  Ronnie T. Moore -->
 <!-- Dynamic 'fix' by: Nannette Thacker -->
 function textCounter(field, countfield, maxlimit) {
-  var fieldval = jQuery(field).attr('value');
+  var fieldval = jQuery(field).val();
+  var countfield = jQuery('input[name="' + countfield + '"]');
```

Comment:
`jshint` tells me that `countfield` is already defined, because it is passed into the function. So you don't need the `var`.